### PR TITLE
Update peer review page based on drafted guidelines

### PR DESCRIPTION
**Pull request by samanthaandrews**
Updated PR review guidelines based on the breakout discussion groups from the 06/11/2025 engineering meeting

[Link to Figjam board](https://www.figma.com/board/p8BeKLxqPrQva59IjdK7pv/Pull-Request--PR--Review-Process-Brainstorm?node-id=0-1&t=x0cRnKj6txvY2qDY-1)

### CODE REVIEW

**Review by nonword — APPROVED** [review]

File: standards/peer-review.md [hunk 1]
```diff
@@ -1,41 +1,38 @@
...
-- Engineers marked as reviewers SHOULD complete the review in a timely manner
-  - This period SHOULD be agreed upon within individual development teams, 24-48 hours has proven to be a reasonable starting place
-  - Code review SHOULD take some time (more than a quick LGTM) but should not take more than 1-2 hours
-  - If a code review is very lengthy or complex, reviewers MAY request that the PR be broken up into smaller pieces and/or request a walkthrough/live review session
- 
-- If changes are requested (or made in response to questions) the requesting engineer SHOULD re-request review when ready
-  - This helps reviewers track the current status of open PRs and plan time for review work
-  - Additional rounds of review SHOULD be completed until the PR can be either merged or closed
-  - To help track PR status teams MAY use GitHub labels (e.g. `Changes Requested`, `Needs Review` and `Ship It`) to help socialize status.
+All pull requests (PRs) **must** be reviewed by at least one other engineer before being merged. Exemptions to this rule require a specific, written policy agreed upon by the team and approved by an Engineering Manager or Director.
```
Comment (EdwinGuzman) [inline standards/peer-review.md:5]: Nit: I think we should follow the readme's key word instruction and write "must" in all-caps.

File: standards/peer-review.md [hunk 2]
```diff
@@ -1,41 +1,38 @@
...
 
-- Engineers SHOULD utilize the Draft PR status to socialize work in progress and to solicit feedback
-  - Reviewers SHOULD NOT use the "request changes" option on draft PR reviews
-  - When a draft PR is ready for full review the requesting engineer MUST update its status and request (or re-request) review from relevant team members
+- **PR reviews are a core responsibility**
+  This isn't an "extra" task; it's a fundamental part of every engineer's job, regardless of level. Actively participating in reviews is crucial for code quality, knowledge sharing, and team success.
```
Comment (EdwinGuzman) [inline standards/peer-review.md:8]: These are great reasons for participating, and they're very much team-oriented. Can we add some points for personal growth? For example, "Active participation will also help developers learn new techniques, improve communication skills, and learn to justify technical decisions."

File: standards/peer-review.md [hunk 3]
```diff
@@ -1,41 +1,38 @@
...
 
-# Peer Review
+- **Be timely & responsive**
+  Prioritize reviews to keep the development flow moving. Aim for initial feedback within a **24-48 hour** timeframe. If you have thoughts but can't give a full review, consider setting up a 30-minute walk-through with the author. After you have pushed updates based on feedback, **re-request a review** from the original reviewers to notify them that the PR is ready for another look. This helps everyone track the status and keeps the process moving.
```
Comment (EdwinGuzman) [inline standards/peer-review.md:11]: Sometimes, the "re-request review" gets lost and many devs miss the notification. Can we add something to the effect of "Or message your team on slack that the PR has been updated with feedback. Do what works on your team!"

File: standards/peer-review.md [hunk 4]
```diff
@@ -1,41 +1,38 @@
...
-- It MAY consist of at least one (1) session of one-on-one peer review.
+- **Verify functionality & tests**
+  Does the code work as expected? Are there adequate tests covering new functionality and edge cases? When necessary, pull down the branch and run the code locally to verify its behavior.
 
-  - Example: One team member would make an appointment with another team member to chat about coding progress, followed by walk-through of code-in-review.
+- **Encourage small, focused PRs**
+  Smaller PRs are easier and faster to review, lead to more thorough feedback, and reduce risk. Reviewers can and should request that large changes be broken down into smaller, logical PRs as a requirement for approval.
 
-- It MAY involve a team member who is less experienced with the tech stack.
+- **Use Draft PRs for early feedback**
```
Comment (EdwinGuzman) [inline standards/peer-review.md:28]: Some teams use labels such as "In Progress" to indicate that a PR is a draft and is still being worked on. There's also "Feedback Wanted". Can we add this as an approach and open it so it's along the lines of "use your team's agreed upon conventions..."?

**Review by EdwinGuzman — COMMENTED** [review]
Thanks! Looks good but some comments.

File: standards/peer-review.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samanthaandrews) [reply standards/peer-review.md:5]: i think it's fine for tech leads to override this process. i'll remove the exemptions to the rule sentence.

**Review by EdwinGuzman — APPROVED** [review]

**Review by alkimcevik — APPROVED** [review]

**Review by chrismulholland — APPROVED** [review]

**Review by yossariano — APPROVED** [review]
Thanks for this- it looks great.

**Review by Toxiapo — APPROVED** [review]

**Review by keithbauer — APPROVED** [review]

**Review by 7emansell — APPROVED** [review]

**Review by charmingduchess — APPROVED** [review]

**Review by bigfishdesign13 — APPROVED** [review]
